packages/core/src/components/tag/tag.tsx

```diff
@@ -26,7 +26,7 @@ export interface ITagProps extends IProps, IIntentProps, React.HTMLAttributes<Ta
      * Click handler for remove button.
      * Button will only be rendered if this prop is defined.
```

Comment:
Can we add a line to the description explaining what the `props` parameter is?

e.g. `Receives the props for the tag that was removed.`